web/server/parser/packageParser.go

```diff
@@ -139,19 +139,17 @@ func (self *outputParser) recordTestingOutcome(outcome string) {
 	self.result.Elapsed = parseDurationInSeconds(fields[2], 3)
 }
 
-var coverageStatementRE = regexp.MustCompile(`coverage: (\d+\.\d)% of statements`)
+var coverageStatementRE = regexp.MustCompile(`coverage: (\d+\.\d)%%? of statements`)
 
 func (self *outputParser) recordCoverageSummary(summary string) {
 	matches := coverageStatementRE.FindStringSubmatch(summary)
-	if len(matches) == 0 {
-		panic("recordCoverageSummary")
-	}
-	parsed, err := strconv.ParseFloat(matches[1], 64)
-	if err != nil {
-		self.result.Coverage = -1
-	} else {
-		self.result.Coverage = parsed
+
+	coverage := float64(-1)
```

Comment:
I think `1.0` will be 64 bit by default?

Reply:
Good point. Done